### ISSUE TITLE
Novo recurso de conjunto: Dicionário

### ISSUE THREAD

**Issue by brenodega28** [basedosdados/website#156]
## Contexto
Depende de #218 

## Tarefas
- [ ] Fazer design
- [ ] Implementar no site
- [ ] Criar botão de Novo Dicionário

**Comment by fernandascovino** [comment]
@rdahis a ideia é virar um novo recurso ne?

**Comment by rdahis** [comment]
Na verdade eu pensei mais e tô afim de transformar os dicionários em metadados a nível de conjunto. Seria bom ter essa discussão com o @d116626 também, porque afetaria o backend também (porque seria bom ter umas funções para interagir com esse metadado).

**Comment by laura-l-amaral** [comment]
Acho que essa discussão de transformar o dicionário em metadado se perdeu né? @rdahis

**Comment by rdahis** [comment]
Toca nessa discussão em https://github.com/basedosdados/mais/issues/1368. Eu criei no nosso banco de metadados a entidade `key`, que é pra ser exatamente esse mapa de chave-valor como metadado. Ainda não usamos na prática.

**Comment by rdahis** [comment]
A ideia era que a 'fonte da verdade' fosse os metadados em `key` e que as tabelas no BQ fossem só uma reflexão dos metadados. Hoje ainda não fazemos assim.

**Comment by laura-l-amaral** [comment]
Certo, mas na implementação, isso não seria mais dificil? Não ficaria mais complicado pro usuário final acessar a informação? Tava conversando com a nay e refletindo sobre as atividades que fizemos no Open Data Day e me parece ter que fazer substituições com dicionário atrapalha bastante o usuário final. Começei a pensar que talvez seja interessante deixar todas as tabelas materializadas no mesmo formato que a tabela de eleições, ou seja, já substituindo as o valor das chaves nas tabelas. Parece viável criar algum script pra ajustar isso via DBT com todas as tabelas que temos

**Comment by rdahis** [comment]
Isso é uma decisão bem não trivial rs. Aumentaria muito nosso storage e a quantidade de GB processados por usuários... Não sei.
